rpm/qt.conf

```diff
@@ -0,0 +1,19 @@
+[Paths]
+Prefix=/usr
+ArchData=share/qt5
```

Comment:
possible to leave defaults and override just those affected by 64bit?

Reply:
The defaults are not what you would expect them to be. I have only set those values which would be wrong if they weren't corrected here.